apps/web/src/services/dashboards/widgets/_configs/config.ts

```diff
@@ -53,9 +53,13 @@ export const ASSET_GROUP_BY = {
     PROVIDER: REFERENCE_TYPE_INFO.provider.key,
     REGION: REFERENCE_TYPE_INFO.region.key,
     // asset reference
-    COMPLIANCE_NUMBER: ASSET_REFERENCE_TYPE_INFO.asset_compliance_number.key,
+    REQUIREMENT_ID: ASSET_REFERENCE_TYPE_INFO.asset_requirement_id.key,
     SERVICE: ASSET_REFERENCE_TYPE_INFO.asset_service.key,
-    COMPLIANCE_TYPE: ASSET_REFERENCE_TYPE_INFO.asset_compliance_type.key,
+    /* HACK:
+    *   The group_by originally matched the reference_key(for autocomplete), but the COMPLIANCE_TYPE is different.
+    *   For now, it is hardcoded, but there's a need to manage it through config in the future.
+    */
+    COMPLIANCE_TYPE: 'cloud_service_type',
```

Comment:
description: key for autocomplete(select dropdown) is `name`, but key for group_by is `cloud_service_type`😭

Reply:
Oh, I think it doesn't matter. That's why we manage group_by config independently.